src/main/java/com/agilethought/internship/sso/services/ServiceApplicationimpl.java

```diff
@@ -20,12 +22,21 @@
 	
 	@Override
 	public UserId createUser(User user) {
+		BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
+		String hashedPassword = passwordEncoder.encode(user.getPassword());
+		System.out.println(user.getPassword());

```

Comment:
Please remove System outs, if a Log is needed use the logger

Reply:
done